### PR TITLE
Features/1/module scaffolding

### PR DESCRIPTION
**Pull request by cam8001**
Added scaffolding, mostly generated with the drupal shell, and using drupal_ti for the base travis file.

### CODE REVIEW

File: akamai.module [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * @file
+ * Contains akamai.module.
+ */
+
+use Drupal\Core\Routing\RouteMatchInterface;
+
+/**
+ * Implements hook_help().
+ */
+function akamai_help($route_name, RouteMatchInterface $route_match) {
```
Comment (nerdstein) [inline akamai.module]: Not sure we need this

File: akamai.module [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      return $output;
+
+    default:
+  }
+}
+
+/**
+ * Implements hook_theme().
+ */
+function akamai_theme() {
```
Comment (nerdstein) [inline akamai.module]: Don't need this.

Can blow away whole module file

**Comment by nerdstein** [comment]
Ready to merge after removal of .module file (unless you think it's wise to leave it there)

**Comment by cam8001** [comment]
No need for it as yet. Removed.

**Pull request merged**